### DISCUSSION TITLE
有没有考虑加一个 onFieldFocus 的 hooks？

### DISCUSSION THREAD

**Discussion by coolbob1998** [alibaba/formily#3243] [Ideas]
* 目前遇到的场景是，可编辑表格最后一列的 Select 的 option 依赖于表格中其他字段
*  表格非 Select 字段变化，基本都会触发最后一列Select 的 option 更新，这样性能较差
*  现在的想法是，只在 onFocus 的时候，计算 Select 的 option ，可以解决问题
*  从 框架层面，加一个 onFieldFocus 的 hooks，处理类似问题是否更简单？

**Comment by janryWang** [comment]
可以onFieldReact里依赖field.active
